<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>1031</width>
      <height>781</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>1001</width>
        <height>671</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <column>
       <property name="text">
        <string>Loan ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Copy ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Catagory</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Publisher</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Loan date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Due date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Tên sách</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tác giả</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Thể loại</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nhà xuất bản</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tìm sách theo:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Nhập nội dung cần tìm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 127);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Tìm sách</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>1021</width>
        <height>611</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <column>
       <property name="text">
        <string>Book Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ISBN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book Category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book Author</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book Publisher</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Available</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>991</width>
        <height>651</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <column>
       <property name="text">
        <string>Loan ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Copy ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Loan date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Due date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Return date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fine</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Reason</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Paid Yet</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nhập mật khẩu hiện tại:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nhập mật khẩu mới:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>300</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nhập lại mật khẩu mới:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>300</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>390</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 127);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Đổi mật khẩu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="butt_t">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Dark Theme</string>
      </property>
     </widget>
     <widget class="QPushButton" name="butt_t_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>120</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Light Theme</string>
      </property>
     </widget>
     <widget class="QPushButton" name="butt_t_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>340</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(63, 66, 172, 255), stop:1 rgba(207, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string>Blue Theme</string>
      </property>
     </widget>
     <widget class="QPushButton" name="butt_t_4">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>340</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 134, 255, 234), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string>Pink Theme</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="butt_u">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hello&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left; 
background-color: rgb(170,170,255);</string>
    </property>
    <property name="text">
     <string> Sách đã/đang mượn</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/calendar.png</normaloff>icon/calendar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_fill">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>15</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_u_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hello&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    text-align: left;
	background-color: rgb(255, 255, 127);
}
QPushButton:hover {
    background-color: rgb(170, 170, 255);
}</string>
    </property>
    <property name="text">
     <string> Tìm sách</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/books.png</normaloff>icon/books.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_u_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hello&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    text-align: left;
	background-color: rgb(255, 255, 127); 
}
QPushButton:hover {
    background-color: rgb(170, 170, 255);
}</string>
    </property>
    <property name="text">
     <string> Phí phạt</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/fine.png</normaloff>icon/fine.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_u_4">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>170</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hello&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    text-align: left;
	background-color: rgb(255, 255, 127);
}
QPushButton:hover {
    background-color: rgb(170, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Đổi mật khẩu</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/user.png</normaloff>icon/user.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_u_5">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>220</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hello&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    text-align: left;
	background-color: rgb(255, 255, 127);
}
QPushButton:hover {
    background-color: rgb(170, 170, 255);
}</string>
    </property>
    <property name="text">
     <string> Đổi Theme</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/themes.png</normaloff>icon/themes.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_u">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>Tên người dùng:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>E-mail:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>  Tổng tiền phạt đang nợ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment: AlignCenter;
background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment: AlignCenter;
background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment: AlignCenter;
background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment: AlignCenter;
background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_55">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>710</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
   <zorder>frame_fill</zorder>
   <zorder>tabWidget</zorder>
   <zorder>butt_u</zorder>
   <zorder>butt_u_2</zorder>
   <zorder>butt_u_3</zorder>
   <zorder>butt_u_4</zorder>
   <zorder>butt_u_5</zorder>
   <zorder>label_u</zorder>
   <zorder>label_u_2</zorder>
   <zorder>label_u_3</zorder>
   <zorder>label_u_4</zorder>
   <zorder>label_u_5</zorder>
   <zorder>label_u_6</zorder>
   <zorder>label_u_7</zorder>
   <zorder>label_u_8</zorder>
   <zorder>pushButton_55</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1302</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
